hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemMkDir.java

```diff
@@ -197,6 +201,23 @@ public void testGetPathPropertyCalledOnMkdir() throws Exception {
 
   }
 
+  @Test
+  public void testGetPathPropertyCalledOnMkdirFallbackDFS() throws Exception {
+    Assume.assumeTrue(getFileSystem().getAbfsStore().getPrefixMode() == PrefixMode.BLOB);
+    AbfsConfiguration abfsConfig = getConfiguration();
```

Comment:
Can you take a clone of object received by `getConfiguration()`. `Mockito.spy()` should help. Or maybe `new Configuration(getRawConfiguration())`.
Reason being more than one test will run in parallel. Setting values on shared object may result bad behavior.

Reply:
Agreed. Will take this. But had also considered initially adding the class to pom to exclude from parallel execution.

Reply:
Also, ingress is not affecting this flow, so not added any new method. But have tested with ingress true and false as well for sanity check.